src/battle_script_commands.c

```diff
@@ -16919,3 +16912,12 @@ void BS_RemoveWeather(void)
     RemoveAllWeather();
     gBattlescriptCurrInstr = cmd->nextInstr;
 }
+
+void BS_ApplyTerastallization(void)
+{
+    NATIVE_ARGS(u8 battler);
```

Comment:
You don't need to pass the battler argument. You can just use `gBattlerAttacker`

Reply:
Yeah, it should could be removed there as well :sweat_smile: